cars/car.js

```diff
@@ -1,9 +1,21 @@
 module.exports = Car
 
-function Car(mpg) {
-
+function Car({ gallons, odometer, trips }) {
+  if (!(this instanceof Car)) {
+    return new Car({ gallons, odometer, trips  });
+  }
+	this.gallons = 0
+	this.odometer = 0
+	this.trips = []
 }
 
 Car.prototype.fill = function(gallons) {
+	this.gallons += gallons;
+}
 
+Car.prototype.drive = function(gallons) {
+	this.gallons = gallons /  this.gallons;
```

Comment:
I think `drive` should accept `miles` not `gallons`
Then the math becomes something like `this.gallons -= miles / this.mpg`